### PR TITLE
Made section for Game Nodes

### PR DESCRIPTION
**Pull request by Kyro3400**
Before anyone comments yes ik this is stupid but this is something to help keep the nodes list organized since we have a lot of nodes then you can see what nodes are used for what and not get confused

### CODE REVIEW

File: bot/discord/serverStatus.js [hunk 1]
```diff
@@ -28,6 +25,11 @@ let nstatus = {
         data: 'Node18'
     }],
 
+    "Game Server Nodes": [{
```
Comment (itzJOHv) [inline bot/discord/serverStatus.js:28]: make this "Gaming Nodes"

File: bot/discord/serverUsage.js [hunk 2]
```diff
@@ -40,6 +37,12 @@ let nstatus = {
         data: 'Node14'
     }],
 
+    "Game Server Nodes": [{
```
Comment (itzJOHv) [inline bot/discord/serverUsage.js:40]: same for this

**Review by itzJOHv — CHANGES_REQUESTED** [review]

File: bot/discord/serverStatus.js (hunk 1 above)
Comment (Kyro3400) [reply bot/discord/serverStatus.js:28]: Ok

**Review by itzJOHv — APPROVED** [review]
looks good to me now

**Comment by Jonfirexbox** [comment]
@danielpmc could you merge this?

**Comment by Kyro3400** [comment]
Dan could you merge this? I fixed it to be like how the embed is now?

**Pull request closed without merging**